.github/workflows/deployment.yaml

```diff
@@ -2,7 +2,7 @@ name: Build and deploy
 
 on:
   push:
-    branches: [develop]
+    branches: [release]
```

Comment:
maybe it would be more convenient to create separate workflows for staging and prod to avoid any conflicts in the future when moving changes between `develop` and `release` branches?

Reply:
Actually, yes, the problem here is that we are going to have significantly different workflows. Will change it appropriately